src/orange/orangeinp/ConvexRegion.cc

```diff
@@ -290,6 +290,48 @@ void Ellipsoid::output(JsonPimpl* j) const
     to_json_pimpl(j, *this);
 }
 
+//---------------------------------------------------------------------------//
+// INFWEDGE
+//---------------------------------------------------------------------------//
+/*!
+ * Construct from a starting angle and interior angle.
+ */
+InfWedge::InfWedge(Turn start, Turn interior)
+    : start_{Turn{std::fmod(start.value(), real_type{1})}}, interior_{interior}
```

Comment:
Is there a good reason to properly handle start values greater than 1? It seems like this would indicate a issue upstream.

Reply:
Good point, the rest of the shapes "validate" that the inputs are correct rather than modifying them. I was thinking of the case where you specify the start angle in `(-1, 0]` but even then `fmod` would let the value remain negative. Fixed.